code/modules/antagonists/devil/devil.dm

```diff
@@ -91,6 +91,7 @@ GLOBAL_LIST_INIT(devil_suffix, list(" the Red", " the Soulless", " the Master",
 	job_rank = ROLE_DEVIL
 	//Don't delete upon mind destruction, otherwise soul re-selling will break.
 	delete_on_mind_deletion = FALSE
+	threat = 30
```

Comment:
Devils are only that threatening when ascended.

Reply:
No excuse for not including scaling since I did for vampires, yeah.